### PR TITLE
Fix issue #140

### PR DESCRIPTION
**Pull request by bwJuniper**
Fix issue #140

### CODE REVIEW

File: apstra/constants.go [hunk 1]
```diff
@@ -9,13 +9,8 @@ const (
 	errResourceConfigureProviderDataSummary   = "Unexpected Resource Configure Type."
 	errResourceConfigureProviderDataDetail    = "Expected '%T', got: '%T'. Please report this issue to the provider maintainers."
 	errDataSourceUnconfiguredSummary          = "Data Source not configured"
-	errDatasourceUnconfiguredDetail           = "Unconfigured data source encountered in Read() operation, possibly because it depends on an unknown value from another object. This leads to weird stuff happening, so we'd prefer if you didn't do that. Thanks!"
 	errProviderInvalidConfig                  = "Provider configuration invalid"
 	errResourceUnconfiguredSummary            = "Resource not configured"
```
Comment (chrismarget-j) [inline apstra/constants.go:13]: `errDataSourceUnconfiguredSummary` and `errResourceUnconfiguredSummary` are now unused.

Please delete these guys as well?

**Review by chrismarget-j — CHANGES_REQUESTED** [review]
Two files in `test` have vanished...

That's okay... I think the whole `test` directory should probably be eliminated.

Please remove the two additional now-superfluous constants?

File: apstra/constants.go (hunk 1 above)
Comment (bwJuniper) [reply apstra/constants.go:13]: I think i got them.  Take a look.

**Review by chrismarget-j — APPROVED** [review]

**Review by bwJuniper — COMMENTED** [review]
i think all changes are ok

**Pull request merged**